Jenkinsfile

```diff
@@ -142,6 +144,45 @@ pipeline {
             }
           }
         }
+        stage('Windows') {
```

Comment:
This stage will run in parallel with the other ones. For clarity, I split it in two substages:
- build
- test

In order to help with compilation and test errors.